### DISCUSSION TITLE
Running the westeros_baseline tutorial

### DISCUSSION THREAD

**Discussion by sharijide** [iiasa/message_ix#488] [Getting started]
Dear Admin, 

I have been encountering issues running the westeros_baseline tutorial.  The second line of the westeros_baseline tutorial python model, that is - **mp = ixmp.Platform()** an error message came-up while running it, it showed: 
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-4-7b00d32da662> in <module>
----> 1 mp = ixmp.Platform()

~\anaconda3\lib\site-packages\ixmp\core.py in __init__(self, dbprops, dbtype, jvmargs)
     76             # if no dbtype is specified, launch Platform with properties file
     77             if dbtype is None:
---> 78                 dbprops = default_dbprops_file() if dbprops is None \
     79                     else find_dbprops(dbprops)
     80                 logger().info("launching ixmp.Platform using config file at '{}'"

~\anaconda3\lib\site-packages\ixmp\default_paths.py in default_dbprops_file()
      5 
      6 def default_dbprops_file():
----> 7     return config.get('DEFAULT_DBPROPS_FILE')
      8 
      9 

~\anaconda3\lib\site-packages\ixmp\config.py in get(key)
     17     if key not in data:
     18         raise RuntimeError(
---> 19             '{} is not configured, do so with `$ ixmp-config -h`')
     20 
     21     return data[key]

RuntimeError: {} is not configured, do so with `$ ixmp-config -h`

The error has hinder me from running other part of the model.

I have tried to use the **ixmp-config** but couldn't get through to the solution.

Help is therefore needed at this point.

Thank you.

**Comment by khaeru** [comment]
This is a question, not an issue, and should be in message_ix, not message-ix-models. I'll attempt to transfer it.
